dbfit-java/build.gradle

```diff
@@ -83,7 +83,10 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task dbConnectionProperties {
```

Comment:
Now that we have both `dbConnectionProperties` and `connectionProperties` - I'm wondering is it clear enough from the name what's the difference between them?